Makefile

```diff
@@ -135,3 +133,25 @@ endif
 .PHONY: up
 up: db  ## Starts the server and associated services with docker-compose, use `clam=1 make up` to run ClamAV
 	@$(BASH_DEV) "rm -f tmp/pids/server.pid && foreman start -m ${FOREMAN_ARG}"
+
+# NATIVE COMMANDS
+.PHONY: native-up
+native-up:
+	bundle install
+	foreman start -m ${FOREMAN_ARG}
+
+.PHONY: native-lint
+native-lint:
+	bundle exec rake lint
+
+.PHONY: native-spec
+native-spec:
+	bundle exec rake spec
+
+.PHONY: native-spec-parallel
+native-spec-parallel:
+	RAILS_ENV=test NOCOVERAGE=true bundle exec rake parallel:spec
+
+.PHONY: native-spec-parallel-setup
+native-spec-parallel-setup:
+	RAILS_ENV=test bundle exec rake parallel:setup
```

Comment:
Have you tested these commands? Can you post some output in the Testing Done section of the PR description?

Reply:
these are all the equivalent native commands that would be run in the docker instance and which are present in the setup documentation

Reply:
@rmtolmach @wayne-weibel These need to be revisited, they don't test anything in the `modules` directory. Also, I'm not sure what the value of these are? What purpose do they serve that isn't accomplished in the existing `lint`, `spec`, `spec_parallel_setup`, `spec_parallel` commands? Is the Makefile the right place for non-Docker commands?

In any case, I'm willing to concede on the existence of the commands themselves as long as these are updated to properly test the `modules` directory so all specs are accommodated

Reply:
I don't use Docker in my vets-api set up, so I'm not super familiar with the use-case for these commands. 

@wayne-weibel, can you respond to Trevor's questions?

Reply:
@bosawt @rmtolmach Why would the Makefile not be the place for commands for running the application?  These commands run the equivalent of the other commands but for a native setup (not running inside of docker-compose); ie. `$(COMPOSE_DEV) $(BASH) ...`.  These do run the `modules` tests, ie. Rakefile

Reply:
Using the Makefile:
<img width="475" alt="image" src="https://github.com/department-of-veterans-affairs/vets-api/assets/42326799/9ab4edb6-1823-407e-9c69-e828c0a24d7c">

Versus using a command to include modules directory:
<img width="724" alt="image" src="https://github.com/department-of-veterans-affairs/vets-api/assets/42326799/b429ad08-95ec-4c3d-a47b-63a983dedd63">

You can see that `native-spec-parallel` is missing 1/3 of the specs (this is because it isn't running tests in the `modules` directories)

Reply:
The reason I am pushing back against the existence of these commands isn't that it's a fundamentally bad idea or anything (I apologize if that was the impression my original comment gave), but that this isn't how Platform has supported or documented usage of the 'native setup'. If Platform is willing to 'own' these commands for support and update documentation to reflect their existence then perhaps they will be useful, otherwise they may just be confusing to new devs. But overall this is a good topic to bring up to the Backend Community of Practice group in Platform, so I'll bring it up to them and let them decide if this is a good addition to adopt.